readme.md

```diff
@@ -66,10 +66,12 @@ $ lint-filter -h
 
   Options:
 
-    -h, --help             output usage information
-    -V, --version          output the version number
-    -f, --format [format]  The output format
-    -b, --branch [branch]  The branch to diff against
+  -h, --help             Output usage information
+  -V, --version          Output the version number
```

Comment:
With the indent change and the capital O's, this doesn't match the output of the command anymore.